front/components/assistant/HelpDrawer.tsx

```diff
@@ -197,13 +197,14 @@ export function HelpDrawer({
             </div>
           </Button.List>
         )}
-        <div className="flex flex-col gap-2 [&>*]:pl-px">
-          <div className="text-base font-bold">Ask questions</div>
+        <div className="flex flex-col gap-4 [&>*]:pl-px">
+          <Page.SectionHeader title="Ask questions" />
           <AssistantInputBar
             owner={owner}
             onSubmit={handleHelpSubmit}
             conversationId={null}
-            actions={[]}
+            hideAttachment={true}
```

Comment:
Here, this code was removed in a previous version but your PR reintroduced it

Reply:
Thank you !